madmom/evaluation/chords.py

```diff
@@ -720,7 +729,7 @@ class ChordEvaluation(object):
         ('undersegmentation', 'UnderSegmentation'),
     ]
 
-    def __init__(self, detections, annotations, name, **kwargs):
+    def __init__(self, detections, annotations, name='', **kwargs):
```

Comment:
This is because in the other classes, I need to create the default string based on other arguments of the function, which cannot be done directly in the method signature. However, I will change this to `name=None` to make it consistent.